src/fides/core/dataset.py

```diff
@@ -356,3 +361,16 @@ def generate_bigquery_datasets(bigquery_config: BigQueryConfig) -> List[Dataset]
         for dataset in bigquery_datasets
     ]
     return unique_bigquery_datasets
+
+
+def generate_dynamo_db_datasets(aws_config: Optional[AWSConfig]) -> Dataset:
+    """
+    Given an AWS config, extract all DynamoDB tables/fields and generate corresponding datasets.
+    """
+    import fides.connectors.aws as aws_connector
```

Comment:
Do we need this here? Looks like we're already importing from `fides.connectors.aws` at the top of the file

Reply:
I think this is a pattern holdover from when we had `boto` as an optional extra on the install (we do this same thing with other `aws` connections) and needed to control import errors.

Will import at the top and create a follow on issue to fix the rest 👍🏽